packages/discovery-react-components/src/components/SearchFacets/components/FacetsGroups/SingleSelectFacetsGroup.tsx

```diff
@@ -65,14 +65,20 @@ export const SingleSelectFacetsGroup: FC<SingleSelectFacetsGroupProps> = ({
     >
       {facets.map(facet => {
         const text = get(facet, facetsTextField, '');
+        const matchingResults = get(facet, 'matching_results');
+        let fullText = text;
```

Comment:
Why not use `text` (instead of a new variable)

Reply:
That value is used both for the `data-key` and buffer below which need a version of the label without the number.